Graphs/Detection of Cycle in directed Graph.cpp

```diff
@@ -0,0 +1,56 @@
+#include<bits/stdc++.h> 
```

Comment:
Try to include a comment containing the **Problem statement** with **Sample Test Cases** with the solutions, that have been tested with the code for future PRs. 👨🏻‍💻
Refer : [CONTRIBUTING.md](https://github.com/akashchouhan16/Data_Structures_And_Algorithms/blob/master/CONTRIBUTING.md "View Guidelines")

Reply:
sorry i wont have testcases and question for it i will create a fresh pr with platform questions and solution sorry for inconvinenece